cdc/owner/feed_state_manager.go

```diff
@@ -140,6 +140,11 @@ func (m *feedStateManager) Tick(state *orchestrator.ChangefeedReactorState) (adm
 	case model.StateStopped, model.StateFailed, model.StateFinished:
 		m.shouldBeRunning = false
 		return
+	case model.StateError:
+		if m.state.Info.Error.IsChangefeedNotRetryError() {
+			m.shouldBeRunning = false
+			return
+		}
```

Comment:
Could you split this change to an other PR?

Reply:
Sure, PTAL: https://github.com/pingcap/tiflow/pull/6310